include/rtc/configuration.hpp

```diff
@@ -27,12 +27,22 @@
 namespace rtc {
 
 struct IceServer {
+	enum class Type { STUN, TURN };
+
+	enum class RelayType { RELAY_TYPE_TURN_UDP, RELAY_TYPE_TURN_TCP, RELAY_TYPE_TURN_TLS };
```

Comment:
Same here, also since you cast it directly to pass it to libnice, I think it'd be safer to specify the values explicitely, like:
```
enum class RelayType { 
    TurnUdp = NICE_RELAY_TYPE_TURN_UDP, 
    TurnTcp = NICE_RELAY_TYPE_TURN_TCP, 
    TurnTls = NICE_RELAY_TYPE_TURN_TLS
};
```

Reply:
OK